packages/rest-api/src/app.ts

```diff
@@ -471,6 +471,116 @@ app.get('/bridgeTxInfo', async (req, res) => {
   }
 })
 
+// Get Synapse Transaction ID
+app.get('/getSynapseTxId', (req, res) => {
+  try {
+    const query = req.query
+    const originChainId = Number(query.originChainId)
+    const bridgeModule = String(query.bridgeModule)
+    const txHash = String(query.txHash)
+
+    if (!originChainId || !bridgeModule || !txHash) {
+      res.status(400).send({
+        message: 'Invalid request: Missing required parameters',
+      })
+      return
+    }
+
+    Synapse.getSynapseTxId(originChainId, bridgeModule, txHash)
+      .then((synapseTxId) => {
+        res.json({ synapseTxId })
+      })
+      .catch((err) => {
+        res.status(400).send({
+          message:
+            'Ensure that your request matches the following format: /getSynapseTxId?originChainId=8453&bridgeModule=SynapseRFQ&txHash=0x4acd82091b54cf584d50adcad9f57c61055beaca130016ecc3798d3d61f5487a',
+          error: err.message,
+        })
+      })
+  } catch (err) {
+    res.status(400).send({
+      message:
+        'Ensure that your request matches the following format: /getSynapseTxId?originChainId=8453&bridgeModule=SynapseRFQ&txHash=0x4acd82091b54cf584d50adcad9f57c61055beaca130016ecc3798d3d61f5487a',
+      error: err.message,
+    })
+  }
+})
+
+// Get Bridge Transaction Status
+app.get('/getBridgeTxStatus', async (req, res) => {
+  try {
+    const query = req.query
+    const destChainId = Number(query.destChainId)
+    const bridgeModule = String(query.bridgeModule)
+    const synapseTxId = String(query.synapseTxId)
+
+    if (!destChainId || !bridgeModule || !synapseTxId) {
+      res.status(400).send({
+        message: 'Invalid request: Missing required parameters',
+      })
+      return
+    }
+
```

Comment:
If we wanted to prevent unnecessary SDK calls in the scenario where Client passes in an unsupported `destChainId`, we can do a check if a valid chainId is outputted from 

CHAINS_BY_ID[query.destChainId] 

from https://github.com/synapsecns/sanguine/blob/api/method-updates/packages/rest-api/src/constants/chains.ts/#L495